pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -406,24 +400,35 @@ func buildTapDeviceMaker(tapName string, isMultiqueue bool, virtLauncherPID int)
 		createTapDeviceArgs = append(createTapDeviceArgs, "--multiqueue")
 	}
 	cmd := exec.Command("virt-chroot", createTapDeviceArgs...)
+
 	return &tapDeviceMaker{
-		virtLauncherSELinuxLabel: virtLauncherSELinuxLabel,
-		virtHandlerSELinuxLabel:  virtHandlerSELinuxLabel,
-		tapMakingCmd:             cmd,
-		tapName:                  tapName,
-		isMultiqueue:             isMultiqueue,
+		tapMakingCmd:    cmd,
+		tapName:         tapName,
+		isMultiqueue:    isMultiqueue,
+		virtLauncherPID: virtLauncherPID,
 	}, nil
 }
 
 func (tmc *tapDeviceMaker) makeTapDevice() error {
-	runtime.LockOSThread()
-	defer func() {
-		_ = selinux.SetExecLabel(tmc.virtHandlerSELinuxLabel)
-		runtime.UnlockOSThread()
-	}()
+	virtHandlerSELinuxLabel := "system_u:system_r:spc_t:s0"
+	virtLauncherSELinuxLabel := ""
+
+	_, selinuxEnabled, err := kvselinux.NewSELinux()
+	if selinuxEnabled {
```

Comment:
Maybe `err == nil && selinuxEnabled`?

Reply:
Right, that makes it consistent with the pattern in https://github.com/kubevirt/kubevirt/blob/94648c8fb52054ae31d1ea2c521f161169655608/cmd/virt-handler/virt-handler.go#L324 .

Done.